packages/picasso.js/src/core/chart-components/labels/labels.js

```diff
@@ -2,15 +2,20 @@ import strategies from './strategies';
 
 /**
  * @typedef {object} ComponentLabels
- * @property {string} [type='labels']
+ * @extends ComponentSettings
  */
```

Comment:
maybe an example?
(to specify the value type should have)

Reply:
All components should have examples (actually most definitions should IMO). I'm just not gonna have the time to fix that in this PR.